src/components/rooms/rawusagedata.ts

```diff
@@ -0,0 +1,10 @@
+import { Group } from '../groups/group'
+
+/**
+ * The number of events in a room on a given day
+ */
+export class RawUsageData extends Group {
+  room: number
+  day: Date
+  many: number | string
```

Comment:
Does this filed represent the number of events for a given room / day? If so I think something like `count` would be a better name for it.

Reply:
And also it should have only type number